<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="6">
     <widget class="QPushButton" name="helpBtn">
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="closeMyComBtn">
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="actionExit">
      <property name="text">
       <string>退出程序</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="7">
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;调试参数输入&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>串口设置</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>串口</string>
        </property>
       </widget>
       <widget class="QComboBox" name="portNameComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
       <widget class="QComboBox" name="baudRateComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>数据位 </string>
        </property>
       </widget>
       <widget class="QComboBox" name="dataBitsComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
       <widget class="QComboBox" name="parityComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
       <widget class="QComboBox" name="stopBitsComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>数据流</string>
        </property>
       </widget>
       <widget class="QComboBox" name="flowTypeComboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>串口设置</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>681</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="chradioButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>字符形式显示</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="hexradioButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>十六进制显示</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="recvbyteslcdNumber">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>20</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="clearUpBtn">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>清空显示</string>
        </property>
       </widget>
       <widget class="QPushButton" name="actionClearBytes">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>计数清零</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>飞行数据</string>
       </attribute>
       <widget class="QGraphicsView" name="standardSixAltimetro">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>341</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="standardSixQADI">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>20</y>
          <width>201</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="standardSixQCompass">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>20</y>
          <width>201</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>数据显示</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="customPlot" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>调试参数输入</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="ContentSelectComboBox"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="openMyComBtn">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="actionSave">
      <property name="text">
       <string>保存数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menubar">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menubar"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>hellWarrior地面站系统</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
